crypto/cms/cms_local.h

```diff
@@ -103,6 +103,12 @@ struct CMS_SignerInfo_st {
     const CMS_CTX *cms_ctx;
     /* Set to 1 if signing time attribute is to be omitted */
     int omit_signing_time;
+    /* Remember which aspects have been verified */
+    int verify_result;
+    int verify_failure;
```

Comment:
Aren't these two flags implied by the three following ones?
If so, remove each one that is not strictly needed,
and for each remaining one add a comment what exactly it means, also to explain why it is needed.